packages/clerk_auth/lib/src/clerk_api/api.dart

```diff
@@ -91,6 +94,21 @@ class Api with Logging {
   ///
   String get domain => _domain;
 
+  /// Sends a map of data to the telemtry endpoint
```

Comment:
Lets extract this function and the static endpoint into its own `ClerkTelemetry` class.